README.md

```diff
@@ -108,16 +108,68 @@ skew may result in `system time has changed` messages in VM logs.
 #### Network
 
 The guest agent uses network interface metadata to manage the network
-interfaces in the guest by performing the following tasks:
-
-*   Enabled all associated network interfaces on boot.
+interfaces by performing the following tasks:
+
+*   Enable all associated network interfaces on boot.
+    *   Detect the current active network manager service that is managing the
+        primary NIC.
+    *   Rollback and delete any files/configurations left behind by all other
+        supported network manager services.
+    *   Write and apply new configurations for the secondary NICs using the
+        network manager service detected in the first step.
 *   Setup or remove IP routes in the guest for IP forwarding and IP aliases
     *   Only IPv4 IP addresses are currently supported.
     *   Routes are set on the primary ethernet interface.
     *   Google routes are configured, by default, with the routing protocol ID
         `66`. This ID is a namespace for daemon configured IP addresses. It can
         be changed with the config file, see below.
 
+On Linux, supported network managers are as follows. These are listed by
+descending priority and include the location at which the configuration files
+are written.
+
+*   `netplan`
+    *   Config location: `/run/netplan/`
+        *   ex: `/run/netplan/20-google-guest-agent-eth0.yaml`
+    *   Dropin location: `/etc/systemd/network/`
+        *   ex: `/etc/systemd/network/10-netplan-eth0.network.d/`
+*   `wicked`
+    *   Config location: `/etc/sysconfig/wicked`
```

Comment:
nit:`/etc/sysconfig/network` instead of `/etc/sysconfig/wicked` need to adjust example path as well

Reply:
Done.